.github/workflows/release-pr.yml

```diff
@@ -35,3 +44,29 @@ jobs:
       uses: ./.github/actions/extract-changelog
       with:
         version: ${{ steps.get_version.outputs.version }}
+
+  tag-release:
+    needs: validate
+    runs-on: ubuntu-latest
+    # Only when the PR is merged!
+    if: github.event.pull_request.merged == true
```

Comment:
is ` == true` needed?

Reply:
Oh right, this is not a string here. Thanks!

Reply:
I am also not sure that this syntax actually works; in my CI battles I remember that comparing with bools was tricky. see here for example: https://github.com/dfinity/motoko/blob/master/.github/actions/performance/action.yml#L28

Reply:
Looks like it is a common trap: https://github.com/actions/runner/issues/1483
and https://stackoverflow.com/questions/76292948/github-action-boolean-input-with-default-value

Reply:
[merged](https://docs.github.com/en/webhooks/webhook-events-and-payloads#pull_request:~:text=mergeable_state%20string-,merged,-boolean%20or%20null) is boolean, so this is fine here.
But you are right that probably all GH Actions have string types